### PR TITLE
Task two

### PR DESCRIPTION
**Pull request by mshepida**

### CODE REVIEW

File: mentoring-app/src/app/header/header.component.html [hunk 1]
```diff
@@ -1 +1,9 @@
-<p>header works!</p>
+<mat-toolbar class="header">
+    <div class="header_logo">
+        <img src="../../assets/images/courses-logo.png" alt="logo">
```
Comment (OksanaHodysh) [inline mentoring-app/src/app/header/header.component.html]: Can be simplified to logo, thanks to default configuration of angular cli (in angular.json) that moves all of our assets into the dist folder.

File: mentoring-app/src/app/footer/footer.component.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.footer {
+    background: white;
+
+    &_text {
+        font-size: 16px;
+    }
+}
```
Comment (OksanaHodysh) [inline mentoring-app/src/app/footer/footer.component.scss]: According to the homework requirements, your footer should be sticky, i.e. always stay at the bottom of the window

Comment (mshepida) [reply mentoring-app/src/app/footer/footer.component.scss]: i gave min-height: 100% to the main content, so it should alway be at the bottom

File: mentoring-app/src/app/header/header.component.html (hunk 1 above)
Comment (mshepida) [reply mentoring-app/src/app/header/header.component.html]: fixed

**Pull request merged**